### ISSUE TITLE
The 'dataclasses' distribution was not found and is required by screeninfo

### ISSUE THREAD

**Issue by kevinsmia1939** [hhannine/superpaper#39]
Hello,

I try to run superpaper on openSUSE Tumbleweed, however, from the error it says that I need dataclasses python module to run superpaper which required by screeninfo.

But what I read on dataclasses, it should have itself included in python 3.7.
https://github.com/ericvsmith/dataclasses

The logs show that python 3.8 is use. So why do I still need dataclasses module to get it to work?
Thanks.

```
kev@linux:~> DESKTOP_SESSION=plasma superpaper
Traceback (most recent call last):
  File "/usr/bin/superpaper", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3252, in <module>
    def _initialize_master_working_set():
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3235, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3264, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'dataclasses' distribution was not found and is required by screeninfo
```

**Comment by hhannine** [comment]
Dataclasses is a requirement of the screeninfo library that I use. I don't know why it isn't available under your 3.8 python environment, or why it wasn't installed by pip as a dependency.

**Comment by kevinsmia1939** [comment]
Alright, will look into it.
